packages/react/package.js

```diff
@@ -42,7 +42,8 @@ read('./dist/react')
 
     if (file === 'dist/react/package.json') {
       ['alias', 'browserslist', 'scripts', 'devDependencies'].forEach(p => delete packageFile[p]);
-      packageFile.peerDependencies = { '@clr/core': packageFile.dependencies['@clr/core'] };
+      // move @clr/core from dependencies to peerDependencies and assign version number that matches the package.json
+      packageFile.peerDependencies = { '@clr/core': packageFile.version };
```

Comment:
how strict should the version be for `@clr/core` here? Currently I have it to match exactly.

The above script will copy the version number in its package.json. For example, if the version number is 4.0.1 for this package it will list:

```
  "peerDependencies": {
    "@clr/core": "4.0.1"
  },
```

Reply:
As long as we are not breaking semver we could probably do something like

```javascript
{ '@clr/core': `^${packageFile.version}` }
```

Reply:
I think that's reasonable. will update.